plugins/abnormal_security/plugin.spec.yaml

```diff
@@ -29,6 +40,8 @@ version_history:
   - '1.0.0 - Initial plugin'
 references:
   - '[Abnormal Security](https://abnormalsecurity.com/)'
+links:
```

Comment:
Links should be this, references should point to API docs that are relevant to the plugin actions